doc/neps/nep-0038-SIMD-optimizations.rst

```diff
@@ -168,6 +170,25 @@ The subjective criteria for accepting new loops are:
 - maintainability: how readable is the code
 - performance: benchmarks must show a significant performance boost
 
+.. _new-intrinsics:
+
+Adding a new intrinsic
+~~~~~~~~~~~~~~~~~~~~~~
+
+If a contributor wants to use a platform-specific SIMD instruction that is not
+yet supported as a universal intrinsic, then:
+
+1. If should be added as a universal intrinsic for all platforms
```

Comment:
typo, "It be should added" as opposed to "if should be added"

Reply:
fixing